games-emulation/bsnes_libretro/bsnes_libretro-0.94.20171227git.recipe

```diff
@@ -46,7 +44,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	for bsnesProfile in "accuracy" "balanced" "performance"; do
-		sed -e "s/@DISPLAY_VERSION@/v${portVersion%.git*}/" \
+		sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
```

Comment:
We'll also have the same problem here as with fbalpha_libretro: `20171227git` < `git20171227` so `pkgman update` would think that `bsnet_libretro-20171227git` is older than `bsnet_libretro-git20171227`.
Users would have to manually uninstall `bsnet_libretro-git20171227`.

We can fix it by renaming the recipe to `bsnes_libretro-0.94_20171227.recipe` and the patchset to `bsnes_libretro-0.94_20171227.patchset`.
```diff
-		sed -e "s/@DISPLAY_VERSION@/v${portVersion%git}/" \
+		sed -e "s/@DISPLAY_VERSION@/v${portVersion/_/-}/" \
```